lighthouse-core/lib/i18n/locales/cs.json

```diff
@@ -824,6 +866,21 @@
   "lighthouse-core/audits/manual/pwa-page-transitions.js | title": {
     "message": "Přechody na jiné stránky nepůsobí zabržděně"
   },
+  "lighthouse-core/audits/maskable-icon.js | description": {
+    "message": "Maskovatelná ikona zajišťuje, že obrázek při instalaci aplikace do zařízení vyplní celý rámec a nevyskytnou se černé okraje. [Další informace](https://web.dev/maskable-icon/)"
+  },
+  "lighthouse-core/audits/maskable-icon.js | failureTitle": {
+    "message": "Manifest nemá maskovatelnou ikonu"
+  },
+  "lighthouse-core/audits/maskable-icon.js | title": {
+    "message": "Manifest má maskovatelnou ikonu"
+  },
+  "lighthouse-core/audits/metrics/cumulative-layout-shift.js | description": {
+    "message": "Kumulativní změna rozvržení je součet všech změn rozvržení, k nimž došlo při načítání stránky. Změna rozvržení je libovolná změna pozice, k níž u prvku dojde poté, co je viditelný uživateli. Všechny změny rozvržení se zaznamenávají a přiděluje se jim skóre. Výsledné kumulativní skóre má hodnotu mezi 0 a 1. 0 znamená perfektně stabilní stránku, skóre >= 0,5 znamená vysoce proměnlivou stránku. [Další informace](https://web.dev/cls)"
+  },
+  "lighthouse-core/audits/metrics/cumulative-layout-shift.js | title": {
```

Comment:
isn't this out-of-date already? 😬 do we need another import to get the correct one for the move to i18n.js `cumulativeLayoutShiftMetric`?

Reply:
hehehehe

Reply:
@exterkamp can we immediately do another import just to get the default locale here fixed?

Reply:
poor translators, I bet the description was a lot of work

Reply:
> @exterkamp can we immediately do another import just to get the default locale here fixed?

Yes. But an automated import will reset all languages to English for that string. So. We can manually commit the new, correct English one and export everything at the same time. This will reset the default, and start translating the new one, without poisoning the translated ones.

Are these ones just so bad that we shouldn't commit them, or that they just aren't perfect and need to be redone immediately (but we could still land this)?

Reply:
I think there are two separate issues going on here @exterkamp.

1. The titles of CLS and LCP were moved out of the metric files into i18n.js. The string didn't change so translations should be fine, but the key did and AFAICT in all languages we'll be using the english name for CLS and LCP right now which would be pretty bad for 6.0 IMO.
2. The *description* of CLS changed to be shorter. I see no need to force a retranslation of all of these before 6.0 if we can still get the longer translated version in. It seems fine if it lags a little bit.

I am mostly concerned about the first bullet.

Reply:
> The titles of CLS and LCP were moved out of the metric files into i18n.js. The string didn't change so translations should be fine, but the key did and AFAICT in all languages we'll be using the english name for CLS and LCP right now which would be pretty bad for 6.0 IMO.

Since the string didn't change, just the key, and we don't send the key to the translators, will these insta-translate if we resubmit, @exterkamp?

Reply:
⬆️ THIS! :)

Thanks @brendankenny for saying what I was horribly trying to get at with "do we need another import to get the correct one" 😆

Reply:
i think we ask @exterkamp  this once a quarter. can we write the answer in the docs